app/composer.json

```diff
@@ -107,12 +107,18 @@
     "phpoffice/phpspreadsheet": "^1.15",
     "predis/predis": "^1.1",
     "composer/composer": "^2.2",
-    "beberlei/doctrineextensions": "^1.3"
+    "beberlei/doctrineextensions": "^1.3",
+    "freshconsulting/wcag_linter": "dev-master",
+    "squizlabs/php_codesniffer": "^3.6"
   },
   "repositories": [
     {
       "type": "git",
       "url": "https://github.com/dennisameling/FOSOAuthServerBundle.git"
+    },
+    {
+      "type": "git",
+      "url": "https://github.com/FreshConsulting/FreshConsulting-WCAG-Linter.git"
```

Comment:
Why can't we use some stable release? This is slow for "composer install" and possibly dangerous if they accidentally merge some buggy or vulnerable PR to their development branch. There is smaller chance of this happening in a stable release.

Reply:
There is no releases in this [repo](https://github.com/FreshConsulting/FreshConsulting-WCAG-Linter). It is the one branch.

Reply:
I see. I crated this issue to ask about releasing it to Packagist: https://github.com/FreshConsulting/FreshConsulting-WCAG-Linter/issues/3